hexa/plugins/connector_dhis2/api.py

```diff
@@ -165,3 +183,15 @@ def fetch_indicators(self):
             "indicators", params={"fields": ":all"}, page_size=100
         ):
             yield [IndicatorResult(data) for data in page["indicators"]]
+
+    def fetch_organisation_units(self):
+        for page in self._api.get_paged(
+            "organisationUnits", params={"fields": ":all"}, page_size=100
+        ):
+            # rewrite path -> replace "/" by "." for correct ltree path
+            # warning: in place edit, can side effect on tests
+            for element in page["organisationUnits"]:
```

Comment:
This rewrite should happen in `sync.py`

Reply:
But let's discuss it, might be ok until we agree on a "fetch & sync" pattern

Reply:
I agree. I hesited a lot where to do that operation, and I propose the following "rule": When doing input validation & cleaning, do it as early as possible.